app/src/app/constants/configuration.ts

```diff
@@ -1,14 +1,32 @@
 import { LngLatLike } from "maplibre-gl";
-import type { MapOptions } from "maplibre-gl";
+import type { MapOptions, StyleSpecification } from "maplibre-gl";
+import * as basemapLayers from "./basemapLayers.json";
 
-const PUBLIC_MAPTILER_API_KEY = "WlatIY6MghFCwInJhBkl";
-const MAPTILER_API = process.env.NEXT_PUBLIC_MAP_API || PUBLIC_MAPTILER_API_KEY;
+const BASEMAP_LAYERS = Array.from(basemapLayers);
 
-export const MAP_TILES = `https://api.maptiler.com/maps/dataviz/style.json?key=${MAPTILER_API}`;
 export const MAP_CENTER: LngLatLike = [-105.358887, 39.113014]; // colorado
 
+export const BASEMAP_STYLE: StyleSpecification = {
+  version: 8,
+  sources: {
+    protomaps: {
+      type: "vector",
+      attribution:
+        '<a href="https://github.com/protomaps/basemaps">Protomaps</a> © <a href="https://openstreetmap.org">OpenStreetMap</a>',
+      url: `pmtiles://${process.env.NEXT_PUBLIC_S3_BUCKET_URL}/basemaps/20240325.pmtiles`,
+    },
+    counties: {
+      type: "vector",
+      url: `pmtiles://${process.env.NEXT_PUBLIC_S3_BUCKET_URL}/basemaps/tiger/tiger2023/tl_2023_us_county_full.pmtiles`,
+    },
+  },
+  layers: BASEMAP_LAYERS,
+  glyphs: `${process.env.NEXT_PUBLIC_S3_BUCKET_URL}/fonts/{fontstack}/{range}.pbf`,
+  sprite: `${process.env.NEXT_PUBLIC_S3_BUCKET_URL}/sprites/white`,
```

Comment:
great!

Reply:
could even support multiple themes by passing the last bit as `sprites/${theme}`!